pages/accountLists/[accountListId]/tools/GetAppeals.graphql

```diff
@@ -0,0 +1,15 @@
+# Write your query or mutation here
+query GetAppeals($accountListId: ID!) {
+  appeals(accountListId: $accountListId, primary: false) {
```

Comment:
This is probably going to need to support pagination eventually. By default the API will only return 25 items.